pkg/net/cidrs/triemap.go

```diff
@@ -272,21 +271,20 @@ func (t *trieMap) getIPv6(addr netip.Addr) (int, bool) {
 				curr = curr.child1
 			} else {
 				// dead end
-				return -1, false
+				break
 			}
 		} else {
 			if curr.child0 != nil {
 				curr = curr.child0
 			} else {
 				// dead end
-				return -1, false
+				break
 			}
 		}
 		// check for a match in the current node
 		if curr.value != nil && curr.value.cidr.Contains(addr) {
 			return curr.value.key, true
 		}
 	}
-	// we walked to the final IP bit without a match
 	return -1, false
```

Comment:
these were unreachable / dead code before, consolidated with the dead-end returns => `break` which we always hit. I would hope it compiles the same ...

Reply:
👍